include/crm_internal.h

```diff
@@ -171,4 +171,10 @@ const char *crm_xml_add_last_written(xmlNode *xml_node);
 void crm_xml_dump(xmlNode * data, int options, char **buffer, int *offset, int *max, int depth);
 void crm_buffer_add_char(char **buffer, int *offset, int *max, char c);
 
+#if defined(WITH_ATTRIBUTE_OUTPUT_ARGS)
```

Comment:
let's use PCMK__ prefix for this and OUTPUT_ARGS (feel free to shorten names, though I don't have any suggestions)

Reply:
Done.  I also pushed a patch to my fosa repo to change what gets defined to PCMK__WITH_ATTRIBUTE_OUTPUT_ARGS.  I don't care so much about the length of the name because it's getting passed around behind the scenes.  We shouldn't have to use it.